docs/configuration/config-file-reference.md

```diff
@@ -771,6 +771,29 @@ lifecycler:
 # After what time a series is considered to be inactive.
 # CLI flag: -ingester.active-series-metrics-idle-timeout
 [active_series_metrics_idle_timeout: <duration> | default = 10m]
+
+global_limits:
+  # Max ingestion rate (samples/sec) that ingester will accept. This limit is
```

Comment:
I think it would be good to document that this is actually an exponentially weighted moving average here, though used for some existing things already.